content/ember/v3/ember-string-prototype-extensions.md

```diff
@@ -0,0 +1,63 @@
+---
+id: ember-string.prototype-extensions
+title: Deprecate String prototype extensions
+until: '4.0.0'
+since: 'Upcoming Features'
```

Comment:
I read the deprecations RFC a bit yesterday. Do the metadata here need to follow the proposed syntax in the RFC?

Reply:
I was thinking about the same thing, but given how we currently present and use the metadata I don't think it's necessary. I have an idea for an improvement to the app, though.